assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile

```diff
@@ -1,4 +1,5 @@
 # FROM quay.io/konveyor/tca-service-api
-FROM quay.io/mihirc36/tackle-container-advisor
+# FROM quay.io/mihirc36/tackle-container-advisor
```

Comment:
We can possibly remove this line completely

Reply:
done